### PR TITLE
Exclude failing tests using vm.continuations flag

### PR DESCRIPTION
**Pull request by fengxue-IS**
java/lang/Thread/virtual/CarrierThreadWaits.java#id0 eclipse-openj9/openj9#18051 java/lang/Thread/virtual/GetStackTrace.java eclipse-openj9/openj9#18052 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#default eclipse-openj9/openj9#18053 serviceability/jvmti/vthread/ThreadStateTest/ThreadStateTest.java eclipse-openj9/openj9#18054

### CODE REVIEW

File: openjdk/excludes/ProblemList_openjdk21-openj9.txt [hunk 1]
```diff
@@ -546,7 +548,9 @@ serviceability/jvmti/vthread/ContinuationTest/ContinuationTest.java https://gith
 serviceability/jvmti/thread/GetFrameCount/framecnt01/framecnt01.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
 serviceability/jvmti/thread/GetStackTrace/getstacktr03/getstacktr03.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
 serviceability/jvmti/thread/GetStackTrace/getstacktr05/getstacktr05.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
+serviceability/jvmti/vthread/ThreadStateTest/ThreadStateTest.java https://github.com/eclipse-openj9/openj9/issues/18054 generic-all
 serviceability/jvmti/vthread/VThreadNotifyFramePopTest/VThreadNotifyFramePopTest.java https://github.com/adoptium/aqa-tests/issues/1297 aix-all
+serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#default https://github.com/eclipse-openj9/openj9/issues/18053 generic-all
 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#no-vmcontinuations https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
```
Comment (babsingh) [inline openjdk/excludes/ProblemList_openjdk21-openj9.txt:554]: Ref: https://github.com/adoptium/aqa-tests/pull/4737#discussion_r1310827632.

re https://github.com/eclipse-openj9/openj9/issues/18053:

`VThreadStackRefTest already has an exclude for the `no-vmcontinuations` variant. You can remove `no-vmcontinuations` to also disable the `default` variant of the test case.
```suggestion
serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
```

Comment (fengxue-IS) [reply openjdk/excludes/ProblemList_openjdk21-openj9.txt]: updated

**Review by babsingh — APPROVED** [review]
@llxia @LongyuZhang Requesting your review.

**Review by llxia — APPROVED** [review]
LGTM

**Review by karianna — APPROVED** [review]

**Pull request merged**